### PR TITLE
hackage: return query as typed in search bar

### PR DESCRIPTION
**Pull request by moonlightdrive**
## Description of new Instant Answer, or changes

Returns the original query (as typed in search bar), with the IA-triggering phrases (haskell, hoogle, or hackage) removed. The previous behaviour returned a uri-encoded string eg "monad%20transformer"

---

Instant Answer Page: https://duck.co/ia/view/hackage

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moonlightdrive** [comment]
oops not sure why the build is failing here D:

File: share/spice/hackage/hackage.js [hunk 1]
```diff
@@ -42,17 +42,15 @@
         var results = api_result.results;
 
         // Get the original query
-        var script = $('[src*="/js/spice/hackage/"]')[0];
-        var source = $(script).attr("src");
-        var query = source.match(/hackage\/([^\/]*)/)[1];
+        var query = DDG.get_query().replace(/\s*(hoogle|haskell|hackage)\s*/gi, '');
```
Comment (moollaza) [inline share/spice/hackage/hackage.js]: It would be simplest to do `searchTerm: decodeURIComponent(query)`

This way no other changes are necessary 👍

**Comment by moollaza** [comment]
This PR is being closed due to inactivity. If you're still interested in finishing, please let me know and I'll gladly re-open it for you!

**Pull request closed without merging**